cmake/OpenCVUtils.cmake

```diff
@@ -1763,15 +1763,15 @@ endif()
 
 macro(ocv_git_describe var_name path)
   if(GIT_FOUND)
-    execute_process(COMMAND "${GIT_EXECUTABLE}" describe --tags --exact-match --dirty
+    execute_process(COMMAND "${GIT_EXECUTABLE}" --work-tree="${path}" describe --tags --exact-match --dirty
       WORKING_DIRECTORY "${path}"
```

Comment:
Why is `WORKING_DIRECTORY` not enough?

Reply:
Thanks, so https://github.com/microsoft/vcpkg/issues/6708 is mostly a vcpckg issue and has to be solved there